### PR TITLE
refactor(common): split inventory from common

### PR DESCRIPTION
**Pull request by sandersms**
Split the inventory from the common folder to allow for adding capabilities along with the inventory.  This also solves the multiple folder issue that was being reported by the inventory file being in the common folder and directed toward an inventory service capability.

### CODE REVIEW

**Comment by glimchb** [comment]
@sandersms please clone [security-skip.yml](https://github.com/opiproject/opi-api/blob/main/.github/workflows/security-skip.yml) and [security.yml](https://github.com/opiproject/opi-api/blob/main/.github/workflows/security.yml) for inventory
we want CI to trigger make

**Comment by glimchb** [comment]
@sandersms this [line](https://github.com/opiproject/opi-api/blob/53642eec45db20013595897f87474a6b2f728056/inventory/v1/inventory.proto#L19)  needs fix 
```
- option go_package = "github.com/opiproject/opi-api/common/v1/gen/go";
+ option go_package = "github.com/opiproject/opi-api/inventory/v1/gen/go";
```

File: inventory/Makefile [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (C) 2022 Intel Corporation
+# Copyright (c) 2022 Dell Inc, or its subsidiaries.
+
+all:
+	rm -rf ./v1/{autogen.md,gen}
+	mkdir -p  ./v1/gen/{go,cpp,python,java}
+
+	docker run --user=$$(id -u):$$(id -g) --rm -v "${PWD}":/defs namely/protoc-all:1.51_2  --lint -d v1 -l go -o ./v1/gen/go/  --go-source-relative --with-gateway
```
Comment (artek-koltun) [inline inventory/Makefile:9]: you could modify Makefile to use the latest googleapis to build, like in other Makefiles
https://github.com/opiproject/opi-api/pull/387/files#diff-d397fabf5af5abbfeebdcba5c3b68879fb33e5bd432ea70a75e26dbf850a848b

**Review by artek-koltun — APPROVED** [review]

File: inventory/Makefile (hunk 1 above)
Comment (glimchb) [reply inventory/Makefile:9]: +1

**Review by glimchb — CHANGES_REQUESTED** [review]
tiny makefile fix to match other makefiles...

File: inventory/Makefile (hunk 1 above)
Comment (sandersms) [reply inventory/Makefile:10]: Updated Makefile to use latest googleapis.

**Review by glimchb — APPROVED** [review]
lgtm

**Pull request merged**